src/common/types.ts

```diff
@@ -5,6 +5,7 @@ namespace ts {
 
     export type ConvertFilesJob = {
         reads: ConvertFile[],
+        schema?: string,
```

Comment:
We can decide to make this non-optional if the ingest-archiver agrees. If the ingest-archiver always outputs a 10x schema version, then we should make this non-optional

Reply:
Implemented with latest commit.